packages/docs-site/docs/ref/style/functions.md

```diff
@@ -1,3265 +1,36 @@
-(e.g. if `s1` should be disjoint from `s2` with margin `padding`).
-
-#### Parameters
-
-| Name                | Type              | Default value |
-| :------------------ | :---------------- | :------------ |
-| `__namedParameters` | [`string`, `any`] | `undefined`   |
-| `__namedParameters` | [`string`, `any`] | `undefined`   |
-| `padding`           | `number`          | `0.0`         |
-
-#### Returns
-
-`IVarAD`
-
-#### Defined in
-
-[contrib/Constraints.ts:522](https://github.com/penrose/penrose/blob/e7fa8b64f/packages/core/src/contrib/Constraints.ts#L522)
-
----
-
-### disjointIntervals
-
-▸ `Static` **disjointIntervals**(`__namedParameters`, `__namedParameters`): `IVarAD`
-
-Make two intervals disjoint. They must be 1D intervals (line-like shapes) sharing a y-coordinate.
-
-#### Parameters
-
-| Name                | Type              |
-| :------------------ | :---------------- |
-| `__namedParameters` | [`string`, `any`] |
-| `__namedParameters` | [`string`, `any`] |
-
-#### Returns
-
-`IVarAD`
-
-#### Defined in
-
-[contrib/Constraints.ts:611](https://github.com/penrose/penrose/blob/e7fa8b64f/packages/core/src/contrib/Constraints.ts#L611)
-
----
-
-### equal
-
-▸ `Static` **equal**(`x`, `y`): `IVarAD`
-
-Require that the value `x` is equal to the value `y`
-
-#### Parameters
-
-| Name | Type     |
-| :--- | :------- |
-| `x`  | `IVarAD` |
-| `y`  | `IVarAD` |
-
-#### Returns
-
-`IVarAD`
-
-#### Defined in
-
-[contrib/Constraints.ts:341](https://github.com/penrose/penrose/blob/e7fa8b64f/packages/core/src/contrib/Constraints.ts#L341)
-
----
-
-### greaterThan
-
-▸ `Static` **greaterThan**(`x`, `y`, `padding?`): `IVarAD`
-
-Require that the value `x` is greater than the value `y` with optional padding `padding`
-
-#### Parameters
-
-| Name      | Type     | Default value |
-| :-------- | :------- | :------------ |
-| `x`       | `IVarAD` | `undefined`   |
-| `y`       | `IVarAD` | `undefined`   |
-| `padding` | `number` | `0`           |
-
-#### Returns
-
-`IVarAD`
-
-#### Defined in
-
-[contrib/Constraints.ts:355](https://github.com/penrose/penrose/blob/e7fa8b64f/packages/core/src/contrib/Constraints.ts#L355)
-
----
-
-### greaterThanSq
-
-▸ `Static` **greaterThanSq**(`x`, `y`): `IVarAD`
-
-Require that the value `x` is greater than the value `y`, with steeper penalty
-
-#### Parameters
-
-| Name | Type     |
-| :--- | :------- |
-| `x`  | `IVarAD` |
-| `y`  | `IVarAD` |
-
-#### Returns
-
-`IVarAD`
-
-#### Defined in
-
-[contrib/Constraints.ts:370](https://github.com/penrose/penrose/blob/e7fa8b64f/packages/core/src/contrib/Constraints.ts#L370)
-
----
-
-### inRange
-
-▸ `Static` **inRange**(`x`, `x0`, `x1`): `IVarAD`
-
-Require that the value `x` is in the range defined by `[x0, x1]`.
-
-#### Parameters
-
-| Name | Type     |
-| :--- | :------- |
-| `x`  | `IVarAD` |
-| `x0` | `IVarAD` |
-| `x1` | `IVarAD` |
-
-#### Returns
-
-`IVarAD`
-
-#### Defined in
+<div v-for="(f, index) in constrDict" :key="index">
```

Comment:
I'm curious, why do you set `:key` here but not in the other two sections?

Reply:
No particular reason. Removing now.